packages/core/content-manager/admin/src/pages/ListView/ListViewPage.tsx

```diff
@@ -334,6 +354,16 @@ const ListViewPage = () => {
           endActions={
             <>
               <InjectionZone area="listView.actions" />
+              <IconButton
+                disabled={isFetching}
+                label={formatMessage({
+                  id: 'content-manager.listView.refresh',
+                  defaultMessage: 'Refresh ontent',
```

Comment:
```suggestion
                  defaultMessage: 'Refresh Content',
```

Unfortunately there is a lot of duplicated code in this file that could probably be refactored. That's not your responsibility on this PR, but you could store this translation in a variable and use it here and in the empty state.